<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWin</class>
 <widget class="QWidget" name="SelectWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(203, 203, 203);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="SelectTb">
    <attribute name="title">
     <string>信息查询</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,1">
         <item>
          <widget class="QRadioButton" name="RegisterRb">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>注册人员信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="DoorRb">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>门锁管理信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SelectPb">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/button.png);</string>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="RegisterWin" name="RegisterTb">
    <attribute name="title">
     <string>信息注册</string>
    </attribute>
   </widget>
   <widget class="MainWindow" name="FaceTb">
    <attribute name="title">
     <string>实时人脸</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RegisterWin</class>
   <extends>QWidget</extends>
   <header>registerwin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainWindow</class>
   <extends>QWidget</extends>
   <header location="global">mainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
